src/ui/components/App.css

```diff
@@ -1,3 +1,10 @@
+.theme-tab-font-size,
+.secondary-toolbox-header,
+.source-tab,
+h2 {
+  font-size: 14px;
+}
```

Comment:
What's this change for?

Reply:
I admit this looks ugly in this format, and will be cleaning it up further. Here's what's up:

We are using tailwind's type ramp, as shown by this middle column:

<img width="391" alt="image" src="https://user-images.githubusercontent.com/9154902/151432558-0d7e985d-2966-48b3-86df-120299f77cf3.png">

And I noticed that our nav tabs were using 15px, which breaks out of our system. So I'm forcing them back down to 14px. In the coming PRs I'm going to clean this up to use our system more cleanly, but for now I'm just forcing to 14px.